packages/flutter/lib/src/widgets/app.dart

```diff
@@ -148,10 +148,14 @@ class WidgetsApp extends StatefulWidget {
          'must have their initial values '
          '(null, null, and the empty list, respectively).'
        ),
-       assert(onGenerateRoute != null || pageRouteBuilder != null,
-         'If onGenerateRoute is not provided, the pageRouteBuilder must be specified '
-         'so that the default handler will know what kind of PageRoute transition '
-         'bo build.'),
+       assert(
+         builder != null ||
```

Comment:
That doesn't work currently (before this change).  If you specify `home`, you have to specify how to do transitions in case there are multiple routes.  Working on updating the docs for that.  

I suppose we could special case just have a home and no other routes, or having a routes list that only contains home (so not ransitions will be required)?

Reply:
For the sake of this PR, I guess we should things as they are. 

A separate PR, that relaxes the constraints on WidgetsApp's parameters, would be useful.